### DISCUSSION TITLE
Newby on 11ty - how to avoid src/README.html goes into dist/README/index.html

### DISCUSSION THREAD

**Discussion by TomFreudenberg** [11ty/eleventy#3163] [Q&A] [answered]
Hi,

I am new on this project but like it very much as a toolkit for our project.

I have not yet find out, how to prevent the "page renames" on build.

In my case, we have a `src/README.html` and want to get a `dist/README.html` but not a `dist/README/index.html`

Thanks for guiding me
Tom

**Comment by bobmonsour** [comment, marked as answer]
In the front matter of the README.html file, add the following:

```
permalink: /README.html
```

This should result in a readme.html in your dist folder rather than a directory with an index.html inside of it. For more info on how to control your urls, see [https://www.11ty.dev/docs/permalinks/](https://www.11ty.dev/docs/permalinks/).

Hope that helps.

**Reply by TomFreudenberg** [reply]
Thanks, got it
